lms/static/js/views/fields.js

```diff
@@ -601,10 +601,25 @@
 
             updateCharCount: function() {
                 var curCharCount;
+                var remainingCharCount;
+                var $charCount = $('.u-field-footer .current-char-count');
                 // Update character count for textarea
                 if (this.options.maxCharacters) {
                     curCharCount = $('#u-field-textarea-' + this.options.valueAttribute).val().length;
-                    $('.u-field-footer .current-char-count').text(curCharCount);
+                    remainingCharCount = this.options.maxCharacters - curCharCount;
+                    if (remainingCharCount < 70 && remainingCharCount > 60) {
+                        $charCount.attr('aria-live', 'polite');
+                        $charCount.attr('aria-atomic', 'true');
```

Comment:
No need to have 2 `attr` calls because the method accepts multiple attributes at once http://api.jquery.com/attr/#setting-several-attrs

Reply:
it gives error when attribute have - in their name like aria-live, aria-polite.